readme_template.md

```diff
@@ -81,7 +81,7 @@ docker run -it (containerid) bash
 
 ### Option 2: Generate it in your own environment
 
-To generate your own amalgamated hosts files you will need Python 3.5 or later.
+To generate your own amalgamated hosts files you will need Python 3.6 or later.
```

Comment:
This is actually a lie. It can run with Python 3.5. We are just not supporting (as in giving support/help) it anymore.

Reply:
Please, I'm testing things...

Reply:
And to answer, no you won't be able to use Python 3.5 after #1920.